<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeWizard</class>
 <widget class="QDialog" name="TypeWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Type Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/buttons/wizard</normaloff>:/buttons/wizard</iconset>
  </property>
  <widget class="QToolButton" name="toolButton_Add_Type">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>9</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Add Item</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/add</normaloff>:/buttons/add</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Filter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>7</y>
     <width>180</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>26</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="toolTip">
    <string>Filter Items</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Icons.qrc">:/mainWIndow/filter</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Delete_Type">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>9</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string>delete item</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/delete</normaloff>:/buttons/delete</iconset>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewTypes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>250</width>
     <height>315</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>35</y>
     <width>250</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_AddedType">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>17</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Added type</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>87</y>
     <width>250</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>324</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/cancel</normaloff>:/buttons/cancel</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>324</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/save</normaloff>:/buttons/save</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>35</y>
     <width>50</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/buttons/forward</normaloff>:/buttons/forward</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>65</y>
     <width>140</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Resulting type path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>324</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/menus/help_help</normaloff>:/menus/help_help</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TypeWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
